README.md

```diff
@@ -61,6 +61,41 @@ optimizt --webp path/to/directory
 optimizt `find . -type f -name '*.jpg'`
 ```
 
+## About “lossy” and “lossless”
```

Comment:
🤔 

I think

> “lossy” vs “lossless”

or

> Differences between “lossy” and “lossless”

will be a bit better. Because it is not common to use “About” at the start of the sentence 🤔